backend/admin_update_task_router.py

```
@@ -0,0 +1,162 @@
+    if not _verify_admin_secret(provided or ""):
+        raise HTTPException(status_code=401, detail="invalid or missing admin secret")
+
+    try:
+        from services.workflow_orchestrator import get_workflow_orchestrator
+        orchestrator = get_workflow_orchestrator()
+    except Exception as exc:
+        log.warning("admin_update_task.router.orchestrator_unavailable: %s", exc)
+        raise HTTPException(
+            status_code=503,
+            detail=f"WorkflowOrchestrator unavailable: {exc}",
+        )
```

Ruff (0.15.17)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)